packages/cli/src/commonlib/telemetry.ts

```diff
@@ -31,22 +32,40 @@ export class CliTelemetryReporter implements TelemetryReporter {
     if (rootPath) {
       this.rootFolder = rootPath;
       this.reporter.setAppRoot(rootPath);
+
+      // add shared properties
+      const fixedProjectSettings = getFixedCommonProjectSettings(rootPath);
+      this.addSharedProperty(TelemetryProperty.ProjectId, fixedProjectSettings?.projectId ?? "");
+      this.addSharedProperty(
+        TelemetryProperty.IsFromSample,
+        fixedProjectSettings?.isFromSample ?? ""
+      );
+      this.addSharedProperty(
+        TelemetryProperty.ProgrammingLanguage,
```

Comment:
I don't know if it's worthy to add programming language as a common property. Do we use this property anywhere for BI purpose?

Reply:
One more question is: will it be only one programming language for one Teams App project?

Reply:
I remember @hund030 suggested to add this in last vteam meeting.
For one project, the language will be selected at first and can not change later.